core/config/project_settings.cpp

```diff
@@ -1416,6 +1416,9 @@ ProjectSettings::ProjectSettings() {
 	GLOBAL_DEF("display/window/energy_saving/keep_screen_on", true);
 	GLOBAL_DEF("display/window/energy_saving/keep_screen_on.editor", false);
 
+	GLOBAL_DEF_BASIC("animation/warnings/check_invalid_track_paths", true);
+	GLOBAL_DEF_BASIC("animation/warnings/check_angle_interpolation_type_conflicting", true);
```

Comment:
Shouldn't this be advanced setting? If you disable it, you need to know what you are doing.

Reply:
This comment was meant for both settings.

Reply:
As for check_invalid_track_paths, I'm not sure the user will know what to expect, but the warning is there, so I think Advanced is ok.